core_impl/src/test/java/com/google/instrumentation/stats/StatsManagerImplTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.instrumentation.stats;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.instrumentation.common.Timestamp;
+import com.google.instrumentation.stats.View.DistributionView;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Tests for {@link StatsManagerImpl}.
+ */
+@RunWith(JUnit4.class)
+public class StatsManagerImplTest {
+
+  @Rule
+  public final ExpectedException thrown = ExpectedException.none();
+
+  private StatsManager statsManager = new StatsManagerImpl();
+
+  @Ignore
+  @Test
+  public void testRegisterAndGetView() throws Exception {
```

Comment:
Removed the Ignore tag. The implementation is not complete yet but this test should pass anyway.